web/src/dialog_widget.js

```diff
@@ -118,6 +119,24 @@ export function launch(conf) {
 
     const $submit_button = $dialog.find(".dialog_submit_button");
 
+    if (conf.update_submit_on_state_change) {
+        const originalValues = {};
+        $(() => {
+            $submit_button.prop("disabled", true);
+            $(".modal__content input").each(function () {
```

Comment:
So, shouldn't this be `:input`?

Reply:
Yes, but in this PR, I have only applied this for the Email change modal, which has only one input field, and its input type is covered by the current `$(".modal__content input")` selector. In the follow-up PR, where I will apply this option to more modals, I will update `$(".modal__content input")` to cover the input types present in those modals as per requirements.

or Should I update $(".modal__content input") to accept all input field types in this PR only?"